### DISCUSSION TITLE
Taiga UI in production

### DISCUSSION THREAD

**Discussion by ghost** [taiga-family/taiga-ui#194] [Q&A] [answered]
Hello,

Please give advice:
can I use Taiga UI in production or wait any longer?

Thanks :-)

**Comment by MarsiBarsi** [comment, marked as answer]
Hey 👋 

I don't see any major problems that we need to fix. So, if you do not wait for something specific by us, you can start using it :) If you do, write us about it and we'll try to implement it as soon as possible 😉

**Comment by ghost** [comment]
Hey,
Thanks for the advice! Received!
1. If possible, add to the page - https://taiga-ui.dev/getting-started, a clarification - what are the objectives of each module: cdk, core,
kit?
2. Also, if possible, add a description of creating / setting up the initial project template.
The bottom line: it seems that you first need to apply a third-party CSS framework, but you have Grid, Row, Col.
Where to get what?

Thanks :-)
